freqtrade/optimize/backtesting.py

```diff
@@ -151,32 +151,33 @@ def _store_backtest_result(self, recordfilename: Path, results: DataFrame,
             logger.info(f'Dumping backtest results to {recordfilename}')
             file_dump_json(recordfilename, records)
 
-    def _get_ticker_list(self, processed: Dict) -> Dict[str, DataFrame]:
+    def _get_ohlcv_as_lists(self, processed: Dict) -> Dict[str, DataFrame]:
         """
-        Helper function to convert a processed tickerlist into a list for performance reasons.
+        Helper function to convert a processed dataframes into lists for performance reasons.
 
         Used by backtest() - so keep this optimized for performance.
         """
         headers = ['date', 'buy', 'open', 'close', 'sell', 'low', 'high']
-        ticker: Dict = {}
-        # Create ticker dict
+        data: Dict = {}
+        # Create dict with data
         for pair, pair_data in processed.items():
             pair_data.loc[:, 'buy'] = 0  # cleanup from previous run
             pair_data.loc[:, 'sell'] = 0  # cleanup from previous run
 
-            ticker_data = self.strategy.advise_sell(
+            dataframe = self.strategy.advise_sell(
```

Comment:
yes, this is a dataframe ... using that as variable name has a certain smell to it for me - we also don't call the integer variable `INT` ... but instead what it does contain.

maybe `df_ohlcv`? or `df_analyzed` ...